.github/workflows/pythonapp.yml

```diff
@@ -26,6 +27,10 @@ jobs:
         flake8 --version
         python -c 'import monai; monai.config.print_config()'
         MYPYPATH=$(pwd)/monai flake8 $(pwd) --count --statistics
+    - name: Type check with mypy statc analysis
+      run: |
+        mypy --version
```

Comment:
do we need to add mypy to requirements-dev.txt?

Reply:
It is included by transitivity from flake8-mypy, but we should probably include it anyway.